crates/glaredb/src/local.rs

```diff
@@ -296,11 +308,11 @@ impl LocalSession {
                 println!("Created new session. New database path: {path}");
                 *self = new_sess;
             }
-            ("\\quit", None) => return Ok(ClientCommandResult::Exit),
+            ("\\quit", None) => std::process::exit(0),
```

Comment:
This prevents history from being saved, hence the `ClientCommandResult`

Reply:
will update this file. Was a bad merge.